MainTel/MainTel-Troubleshooting/MTM2-Troubleshooting/MTM2_recovery.rst

```diff
@@ -0,0 +1,306 @@
+
+#. Restart control system.
+    .. warning::
+        This step must not be skipped. 
+        Restarting the control system acts as a catch-all for resetting issues. Failing to do so may also cause issues with telemetry.
+
+    #. **Connect** to the admin user on M2 cRIO controller via ssh using the username and password found in `1password`_ Main Telescope.
+
+        .. _`1password`: https://lsstit.1password.com/signin
+
+        .. code-block:: shell
+
+            ssh admin@m2-crio-controller01.cp.lsst.org
+
+    #. **Stop the control system** and wait 3 minutes using the command:
+    
+        .. code-block:: shell
+
+            /etc/init.d/nilvrt stop
+
+    #. **Start the control system** and wait 3 minutes using the command: 
+
+        .. code-block:: shell
+            
+            /etc/init.d/nilvrt start
+        
+        You may press enter to regain your shell prompt when you see the following "Welcome to LabVIEW Real-Time 18.0".
+
+        .. figure:: _static/MTM2_recovery_restart_control_system.png
+            :name: Restarting MTM2 control system
+            :width: 600
+
+    .. _MTM2-recovery-Reset-the-M2-interlock-signal: 
+
+#. `Reset the M2 interlock signal`_ in GIS main cabinet on the second floor, even if the state is "OK".
+
+    .. _`Reset the M2 interlock signal`: https://ts-m2gui.lsst.io/error-handling/error-handling.html#lsst-ts-m2gui-error-reset-m2-interlock-signal
+
+    .. Important::
+
+        Rarely, if ever, will the GIS show the M2 Actuator interlock in any state other than OK. This is even true when it is not OK.
+
+#. Use python EUI/GUI to change MTM2 to **Close Loop** status:
+
+    #. Open the **MTM2 EUI**. Follow instructions in `How to access the MTM2 EUI`_.
+
+    #. Establish **local control** with the EUI by hitting connect, then local (local may be greyed out after hitting connect, this is ok).
+
+        .. figure:: _static/MTM2_recovery_GUI_open_connect.png
+            :name: MTM2 GUI open and connect
+            :width: 600
+            :align: center
+
+    #. Pull up the **Overview widget** by double-clicking on Overview in the list at the bottom of the EUI.
+
+        .. figure:: _static/MTM2_recovery_GUI_overview.png
+            :name: MTM2 GUI Overview
+            :width: 600
+            :align: center
+
+        #. Observe the **Enabled Faults Mask**. It should **not be 0**. If it is, repeat :ref:`reset GIS signal <MTM2-recovery-Reset-the-M2-interlock-signal>`.
+
+            .. note::
+                
+                It is ok if the ``isInterlockEngaged`` indicator is red.  
+
+                .. the original said isInterlockEnabled
+
+        #. You may also look at **Alarms/Warnings** widget to see active alarms (red) or warnings (yellow). If active, reset them with **Reset All Items**. Note: *Make sure you have removed the fault condition*.
```

Comment:
Reset All Items is a button but also can get strongly emphasized with doubled *, your choice!